opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -361,6 +365,9 @@ def __init__(self, context: "SpanContext") -> None:
     def get_context(self) -> "SpanContext":
         return self._context
 
+    def is_recording_events(self) -> bool:
```

Comment:
Oh, this was missing from the no-op implementation? It seems we are severely lacking unit tests for it.

Reply:
Yeah, and not just here. We can revisit after https://github.com/open-telemetry/opentelemetry-python/pull/128.